src/main/resources/db/changelog/ddl/0015_refactor_anonymous_elector_document_tables.xml

```diff
@@ -0,0 +1,190 @@
+        </sql>
+
+        <rollback>
+            <dropTable tableName="anonymous_elector_document_status"/>
+        </rollback>
+    </changeSet>
+
+    <changeSet author="neil.massey@valtech.com" id="0015-EIP1-4179-drop-aed-print-request-tables" context="ddl">
+        <dropTable tableName="aed_print_request_status"/>
+        <dropTable tableName="aed_print_request"/>
+        <rollback>
+            <createTable tableName="aed_print_request">
+                <column name="id" type="uuid">
+                    <constraints nullable="false" primaryKey="true" unique="true"
+                                 primaryKeyName="print_request_pkey"/>
+                </column>
+                <column name="aed_id" type="uuid">
+                    <constraints nullable="false"
+                                 foreignKeyName="fk_print_request_aed"
+                                 references="anonymous_elector_document(id)"/>
+                </column>
+
+                <column name="date_created" type="timestamp">
+                    <constraints nullable="false"/>
+                </column>
+                <column name="created_by" type="varchar(255)">
+                    <constraints nullable="false"/>
+                </column>
+                <column name="version" type="bigint">
+                    <constraints nullable="false"/>
+                </column>
+            </createTable>
+
+            <createIndex indexName="aed_electoral_roll_number_idx" tableName="aed_print_request" unique="false">
+                <column name="electoral_roll_number"/>
+            </createIndex>
+
+            <createTable tableName="aed_print_request_status">
+                <column name="id" type="uuid">
+                    <constraints nullable="false" primaryKey="true" unique="true"
+                                 primaryKeyName="aed_print_request_status_pkey"/>
+                </column>
+                <column name="aed_print_request_id" type="uuid">
+                    <constraints nullable="false"
+                                 foreignKeyName="fk_aed_print_request_status_aed_print_request"
+                                 references="aed_print_request(id)"/>
+                </column>
+                <column name="status" type="varchar(50)">
+                    <constraints nullable="false"/>
+                </column>
+                <column name="event_date_time" type="timestamp">
+                    <constraints nullable="false"/>
+                </column>
+                <column name="date_created" type="timestamp">
+                    <constraints nullable="false"/>
+                </column>
+                <column name="created_by" type="varchar(255)">
+                    <constraints nullable="false"/>
+                </column>
+                <column name="version" type="bigint">
+                    <constraints nullable="false"/>
+                </column>
+            </createTable>
```

Comment:
Not sure how we'd recover the previous data, but that's currently not a concern in prod 👍

Reply:
Yeah, good point - I guess it would still be in the new `anonymous_elector_document_status` table.  Rollbacks where data is concerned is likely to need manual intervention - luckily we don't have any data yet 👍